features/sketch/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id(libs.plugins.wire.android.library.get().pluginId)
+    id("org.jetbrains.kotlin.android")
```

Comment:
This is unnecessary, as we already apply this plugin when applying [`AndroidLibraryConventionPlugin`](https://github.com/wireapp/wire-android-reloaded/blob/52decf5ace84d1aad8f3fcb99c3e35decfb65e70/build-logic/plugins/src/main/kotlin/AndroidLibraryConventionPlugin.kt#L29), in the line above.

Reply:
Good catch!